src/site/content/en/blog/advancing-framework-ecosystem-cds-2019/index.md

```diff
@@ -0,0 +1,208 @@
+    [webpack](https://webpack.js.org/))
++   Server-side rendering
++   Mechanisms to fetch data at a per-page level
++   Encapsulated styling (with [styled-jsx](https://github.com/zeit/styled-jsx))
+
+Next.js optimizes for reduced bundle sizes, and the Chrome team helped identify areas where we could
+help further improve performance. You can learn more about each of them by viewing their requests
+for comments (RFCs) and pull requests (PRs):
+
+1.  An improved webpack chunking strategy that emits more granular bundles, reducing the
+    amount of duplicate code fetched through multiple routes
+    ([RFC](https://github.com/zeit/next.js/issues/7631),
+    [PR](https://github.com/zeit/next.js/pull/7696)).
+2.  Differential loading with the
+    [module/nomodule pattern](../serve-modern-code-to-modern-browsers/#use-lessscript-type%22module%22greater)
+    which can reduce the total amount of JavaScript in Next.js apps by up to 20% with no code
+    changes ([RFC](https://github.com/zeit/next.js/issues/7563),
+    [PR](https://github.com/zeit/next.js/pull/7704)).
+3.  Improved performance metric tracking which utilizes the User Timing API
+    ([PR](https://github.com/zeit/next.js/pull/8069)).
+
+<figure class="w-figure w-figure--center">
+  <img src="barnebys.png" class="w-screenshot" alt="Homepage of Barnebys.com">
+  <figcaption class="w-figcaption">
+    <a href="https://www.barnebys.com/">Barnebys.com</a>, a large search engine for antiques and collectibles, saw a 23% reduction in total JavaScript with granular chunking enabled
+  </figcaption>
+</figure>
+
+We are also exploring other features to improve both the user and developer experience of using
+Next.js, such as:  
+
++   Enabling concurrent mode to unlock progressive or partial hydration of components. 
++   A webpack based conformance system that analyzes all source files and generated assets to
+    surface better errors and warnings ([RFC](https://github.com/zeit/next.js/issues/9310)).
+
+<figure class="w-figure w-figure--center">
+  <img src="conformance.png" class="w-screenshot" alt="Example of a conformance build error in Next.js">
+  <figcaption class="w-figcaption">
+    An example of a conformance build error in Next.js (prototype)
+  </figcaption>
+</figure>
+
+## Nuxt.js
+
+[Nuxt.js](https://nuxtjs.org/) is a web framework that combines Vue.js with different libraries to
+provide an opinionated setup. Similar to Next.js, it includes many features out-of-the-box:
+
++   Routing with default code-splitting
++   Compilation and bundling (using [Babel](https://babeljs.io/) and
+    [webpack](https://webpack.js.org/))
++   Server-side rendering
++   Asynchronous data fetching for every page
++   Default data store ([Vuex](https://vuex.vuejs.org/guide/))
+
+Along with working directly on improving the performance of different tools, we've expanded the
+framework fund to provide monetary support to more open-source frameworks and libraries. With our
```

Comment:
```suggestion
[framework fund](https://opencollective.com/chrome) to provide monetary support to more open-source frameworks and libraries. With our
```

Ignore this if I'm wrong, but this seems to be the first mention of `framework fund`, so this is where we should link to more info